charts/kueue/templates/certmanager/certificate-metrics.yaml

```diff
@@ -1,18 +1,20 @@
 {{- if .Values.enableCertManager }}
+# The following manifests contain a metrics certificate CR.
+# More document can be found at https://docs.cert-manager.io
```

Comment:
/hold

We have a lot of redundant comments. I've created a separate pull request to automatically remove it.

https://github.com/kubernetes-sigs/kueue/pull/5930

Reply:
As the PR is merged, @mbobrovskyi please rebase the PR

Reply:
Done

Reply:
/unhold